src/store/app.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+const initialState = {
+  modalIsOpen: false,
+  modalIsLocked: false,
+};
+
+const app = createSlice({
```

Comment:
Why called app and not modal?

Reply:
I thought about storing here all general app's states, but it can be, as you say, only for the modal.
I will rename it